tests/drivers/stepper/drv8424/api/boards/native_sim.overlay

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Navimatix GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/gpio/gpio.h>
+
+
+/ {
+
+	drv8424: drv8424 {
+		status = "okay";
+		compatible = "ti,drv8424";
+
+
+		dir-gpios = <&gpio1 0 0>; /* D3 */
+		step-gpios = <&gpio1 1 0>; /* D4 */
+		sleep-gpios = <&gpio2 0 GPIO_ACTIVE_LOW>; /* D2 */
+		en-gpios = <&gpio2 1 0>; /* 5 */
+		m0-gpios = <&gpio3 0 0>;
+		m1-gpios = <&gpio3 1 0>;
+		counter = <&counter0>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+		#stepper-motor-cells = <0>;
+
+	};
+
+
```

Comment:
same, you can use them between the gpio nodes if you really want to, there's normally one there :-)